releasenotes/notes/add-usage-hf-api-chat-streaming-91fd04705f45d5b3.yaml

```
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    In the HuggingFaceAPIChatGenerator request additional streaming chunk that contains usage data.
+    Process usage streaming chunk to add usage meta data to the returned ChatMessage.
```

We could emphasize the impact on the user. For example, something like the following:
```suggestion
    When using HuggingFaceAPIChatGenerator with streaming, the returned ChatMessage now contains the number of prompt tokens and completion tokens in its meta data.
    Internally, the HuggingFaceAPIChatGenerator requests an additional streaming chunk that contains usage data.
    It then processes the usage streaming chunk to add usage meta data to the returned ChatMessage.
```